Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -80,8 +81,15 @@ struct DashboardView: View {
                     ProgressView("Loading...")
                 } else {
                     if viewModel.events.isEmpty {
-                        Text("Add your first maintenance",
-                             comment: "Placeholder text for empty maintenance list prompting the user to add a maintenance event") // swiftlint:disable:this line_length
+                // Blank Dashboard view starts
```

Comment:
don't add start and end comments, it doesn't add anything to the codebase `ContentUnavailableView` is clear what this is for